modules/sublimeBidAdapter.js

```diff
@@ -1,135 +1,255 @@
+};
 
-export const spec = {
-  code: BIDDER_CODE,
-  aliases: [],
+/**
+ * Set a new state
+ * @param {Object} value
+ */
+function setState(value) {
+  Object.assign(state, value);
+  log('State has been updated :', state);
+}
+
+/**
+ * Send pixel to our debug endpoint
+ * @param {string} eventName - Event name that will be send in the e= query string
+ */
+function sendEvent(eventName, isMendatoryPixel) {
+  let shoudNotSendPixel = !(isMendatoryPixel || state.debug);
+  let eventObject = {
+    tse: Date.now(),
+    z: state.zoneId,
+    e: eventName,
+    src: 'pa',
+    trId: state.transactionId,
+    ver: SUBLIME_VERSION,
+  };
+
+  let queryString = url.formatQS(eventObject);
+  if (shoudNotSendPixel) {
+    log('Not sending pixel for event (use debug: true to send it): ' + eventName, eventObject);
+  } else {
+    log('Sending pixel for event: ' + eventName, eventObject);
+    utils.triggerPixel('https://' + SUBLIME_ANTENNA + '/?' + queryString);
+  }
+}
+
+/**
+ * Determines whether or not the given bid request is valid.
+ *
+ * @param {BidRequest} bid The bid params to validate.
+ * @return {Boolean} True if this is a valid bid, and false otherwise.
+ */
+function isBidRequestValid(bid) {
+  return !!bid.params.zoneId;
+}
+
+/**
+ * Make a server request from the list of BidRequests.
+ *
+ * @param {BidRequest[]} validBidRequests An array of bids
+ * @param {Object} bidderRequest - Info describing the request to the server.
+ * @return ServerRequest Info describing the request to the server.
+ */
+function buildRequests(validBidRequests, bidderRequest) {
+  window.sublime = window.sublime ? window.sublime : {};
+
+  let commonPayload = {
+    sublimeVersion: SUBLIME_VERSION,
+    // Current Prebid params
+    prebidVersion: '$prebid.version$',
+    currencyCode: config.getConfig('currency.adServerCurrency') || DEFAULT_CURRENCY,
+    timeout: typeof bidderRequest !== 'undefined' ? bidderRequest.timeout : config.getConfig('bidderTimeout'),
+  };
+
+  // RefererInfo
+  if (bidderRequest && bidderRequest.refererInfo) {
+    commonPayload.referer = bidderRequest.refererInfo.referer;
+    commonPayload.numIframes = bidderRequest.refererInfo.numIframes;
+  }
+  // GDPR handling
+  if (bidderRequest && bidderRequest.gdprConsent) {
+    commonPayload.gdprConsent = bidderRequest.gdprConsent.consentString;
+    commonPayload.gdpr = bidderRequest.gdprConsent.gdprApplies; // we're handling the undefined case server side
+
+    // Injecting gdpr consent into sublime tag
+    window.sublime.gdpr = (typeof window.sublime.gdpr !== 'undefined') ? window.sublime.gdpr : {};
+    window.sublime.gdpr.injected = {
+      consentString: bidderRequest.gdprConsent.consentString,
+      gdprApplies: bidderRequest.gdprConsent.gdprApplies
+    };
+  }
+
+  return validBidRequests.map(bid => {
+    let bidHost = bid.params.bidHost || DEFAULT_BID_HOST;
+    let callbackName = (bid.params.callbackName || DEFAULT_CALLBACK_NAME) + '_' + bid.params.zoneId;
```

Comment:
Is it really a param ?

Reply:
Not in the documentation but at least can be change in case of conflicts